.github/lighthouse/pages.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
I couldn't figure out what this file was for? Or if it was important.

Reply:
that's fine, it was committed accidentally, it's a thought experiment about possible config file format, and `lighthouseci.json` solves this.